R/utilities.R

```diff
@@ -3094,7 +3094,8 @@ get_gambl_colours = function(classification = "all",
 get_bams = function(this_sample_id,
                     this_patient_id){
 
-  meta = get_gambl_metadata(tissue_status_filter = c("tumour", "normal"), seq_type_filter = "genome")
+  meta = get_gambl_metadata(tissue_status_filter = c("tumour", "normal"), seq_type_filter = seq_type_filter)
```

Comment:
I think here and in the next line the `seq_type_filter` is not defined for this function, we probably need to add it as the function parameter